test/sharness/t0040-add-and-cat.sh

```diff
@@ -345,6 +345,27 @@ test_add_cat_file() {
     echo "added Qmf35k66MZNW2GijohUmXQEWKZU4cCGTCwK6idfnt152wJ hello2.txt" >> expected &&
     test_cmp expected actual
   '
+
+  test_expect_success "ipfs add with multiple files of same name succeeds" '
+    mkdir -p mountdir/same-file/ &&
+    cp mountdir/hello.txt mountdir/same-file/hello.txt &&
+    ipfs add mountdir/hello.txt mountdir/same-file/hello.txt >actual &&
+    rm mountdir/same-file/hello.txt  &&
+    rmdir mountdir/same-file
+  '
+
+  test_expect_success "ipfs add with multiple files of same name output looks good" '
+    echo "added QmVr26fY1tKyspEJBniVhqxQeEjhF78XerGiqWAwraVLQH hello.txt" >expected &&
```

Comment:
Shouldn't we expect 2 output lines if we add 2 files?
Did you try to have the 2nd file with different content?

Reply:
Will take another look at the test this week. Thanks for flagging.